common/cloudbuild.yaml

```
@@ -0,0 +1,39 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# GCB config for generating docker images.
+#
+# Parameters
+# _IMAGE_NAME: Name of the image. E.g. python-multi
+# _IMAGE_SOURCE_PATH: Path to Dockerfile. E.g. python/googleapis/python-multi
+
+timeout: 7200s  # 2 hours
+steps:
+- name: gcr.io/cloud-builders/docker
+  args: [
+    'build',  '--cache-from',
+    'us-central1-docker.pkg.dev/cloud-sdk-release-custom-pool/release-images-dev/${_IMAGE_NAME}',
```

```suggestion
    'us-central1-docker.pkg.dev/$PROJECT_ID/release-images-dev/${_IMAGE_NAME}',
```

Should this be parameterized or use the built-in `$PROJECT_ID` substitution? (here and elsewhere)

---

Thanks for the feedback. I initially thought that a Louhi project would deal with only a single GCB project, but on second thought, I don't think that's strictly necessary. I cannot use the built-in $PROJECT_ID, because that results in Louhi substituting it with louhi-prod-1. I've parameterized it instead.